project_report/tex/requirements.tex

```diff
@@ -8,6 +8,34 @@ \subsubsection{Functional \& Non-Functional Requirements}
 
 \subsubsection{The Physical Setup (The Brewery Machine)}
 
+The machine is able to produce different kinds of beer depending on the recipe
+used. The beer is produced in batches which have their own batch id. When 
+producing a batch the recipe, quantity, speed and batch id can be configured by 
+an external system. When a batch has been produced the quality inspection system 
+measures the quality for future use. These values are accessible in the machines 
+SCADA system. The SCADA system monitors and supervises both the machines PLC 
+system and the internal sensors PLC system as well as the quality inspection 
+system.
+
+\myparagraph{Sensors}
+To ensure the most optimal environment beer production and machine longevity the 
```

Comment:
Do we care about the machines longevity? 
What the previous comment suggested was to specify why data from the environment was relevant. 
This is only because it effects the beer production.